### PR TITLE
refactor: update readmes

### LINKED ISSUE #635: Ensure the readme of each tool in packages is complete

### PR DESCRIPTION
**Pull request by davifantasia**
Closes #635 

* Updated readmes of
  - commercetools-docs-kit
  - gatsby-theme-docs
  - gatsby-theme-api-docs
  - gatsby-transformer-raml
  - ramldoc-generator
* Removed change-log from test sites
* Removed unused dependencies from test sites

### CODE REVIEW

File: websites/api-docs-smoke-test/CHANGELOG.md [hunk 1]
```diff
@@ -1,158 +0,0 @@
-# @commercetools-website/api-docs-smoke-test
```
Comment (emmenko) [inline websites/api-docs-smoke-test/CHANGELOG.md:1]: Why did you remove this?

File: websites/docs-smoke-test/CHANGELOG.md [hunk 2]
```diff
@@ -1,205 +0,0 @@
-# @commercetools-website/docs-smoke-test
```
Comment (emmenko) [inline websites/docs-smoke-test/CHANGELOG.md:1]: Same for this

File: websites/api-docs-smoke-test/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@commercetools-website/api-docs-smoke-test",
-  "version": "3.0.18",
+  "version": "1.0.0",
```
Comment (emmenko) [inline websites/api-docs-smoke-test/package.json:3]: Why changing the versions? Those are managed by changesets

**Review by emmenko — CHANGES_REQUESTED** [review]

File: websites/api-docs-smoke-test/package.json (hunk 3 above)
Comment (davifantasia) [reply websites/api-docs-smoke-test/package.json:3]: > Why changing the versions? Those are managed by changesets

From one of my discussions with @nkuehn I understand we don't do releases for test sites so that's why I removed everything associated with that.

Comment (emmenko) [reply websites/api-docs-smoke-test/package.json:3]: Yes but it's part of the monorepo/workspace and so changeset will include that as well. It's not a problem in general, those packages are not published anyway.

Revert those changes, as otherwise changeset will regenerate them again next time.

File: packages/gatsby-theme-docs/README.md [hunk 4]
```diff
@@ -37,54 +37,73 @@ The project structure should contain at least the following files and folders:
...
-      {
-        resolve: '@commercetools-docs/gatsby-theme-docs',
-        options: {
-          websiteKey: 'change-me',
-        },
-      },
-    ],
-  };
-  ```
+- `gatsby-config.js`: this is required for a Gatsby website and should contain the website specific configuration. Note that all plugins and associated options must be passed using `configureThemeWithAddOns`, in addition there is no need to pass `gatsby-theme-docs` as a plugin but only the required options. See the code sample below for more details.
```
Comment (emmenko) [inline packages/gatsby-theme-docs/README.md:40]:
```suggestion
- `gatsby-config.js`: this is required for a Gatsby website and should contain the website specific configuration. If your website requires the usage of [add-ons](#using-theme-with-add-ons), you need to configure the main theme using the `configureThemeWithAddOns` helper function. See examples below.
```

**Review by emmenko — COMMENTED** [review]
Looking good. Just left some small suggestions.

**Review by emmenko — APPROVED** [review]
Thanks
